converters/toZigbee.js

```diff
@@ -2125,6 +2125,29 @@ const converters = {
             sendTuyaCommand(entity, 263, 0, [1, value==='LOCK' ? 1 : 0]);
         },
     },
+    
+    // Centralite 3400-D Keypad Alarm
+    3400_arm_mode: {
```

Comment:
Good, can you also make a pr for the docs? otherwise I think it will  be hard for other users on how to use this device (https://github.com/Koenkk/zigbee2mqtt.io/tree/master/docs/devices)

Reply:
Sure, i will do this. But i think we need some help in finding out what we have to send to the device, so the device can set the wanted mode.

Reply:
Have you made any progress here?

Reply:
I need some help. Do i have to pair the device to the original base and then sniff the traffic with wireshark?

Reply:
Yes

Reply:
I've ordered a cc2531 kit to flash and sniff the traffic. I will report soon.

Reply:
I'am now able to sniff the traffic. I captured for example the **disarm** process:

**Disarming:**
![grafik](https://user-images.githubusercontent.com/64753405/84828174-e75afd80-b025-11ea-9f91-1d84c0c884bb.png)

**Response:**
![grafik](https://user-images.githubusercontent.com/64753405/84828256-11142480-b026-11ea-82bc-f9fb4116e1e3.png)


Were these the informations we need to add support for the device for zigbee2mqtt?

Reply:
`entity.commandReponse('ssIasAce', 'armRsp', {armnotification: 0})`

Reply:
@Koenkk sorry, but i can't do much with this information. :/

Reply:
Change the converter to 

```js
3400_arm_mode: {
    key: ['disarm],
    convertSet: async (entity, key, value, meta) => {
        entity.commandReponse('ssIasAce', 'armRsp', {armnotification: 0})
    },
},    
```

Now you can send the command by sending to topic `zigbee2mqtt/[DEVICE_FRIENDLY_NAME]/set` with payload `{"disarm":""}`

Reply:
Thank you. I added your changes to `toZigbee.js` and edited in `devices.js` in the device section `toZigbee` accordnigly

First i got the following error:
```
/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/converters/toZigbee.js:1323
    3400_arm_mode: {
    ^^^^

SyntaxError: Invalid or unexpected token
```

I've changed `3400_arm_mode:` to `centralite_arm_mode:`

That worked and after sending the payload via mqtt i've got:

```
zigbee2mqtt:debug 2020-06-17 19:55:55: Received MQTT message on 'zigbee2mqtt/0x000d6f000d8a5f63/set' with data '{"disarm":""}'
zigbee2mqtt:debug 2020-06-17 19:55:55: Publishing 'set' 'disarm' to '0x000d6f000d8a5f63'
zigbee2mqtt:error 2020-06-17 19:55:55: Publish 'set' 'disarm' to '0x000d6f000d8a5f63' failed: 'TypeError: entity.commandReponse is not a function'
zigbee2mqtt:debug 2020-06-17 19:55:55: TypeError: entity.commandReponse is not a function
    at Object.convertSet (/opt/zigbee2mqtt/node_modules/zigbee-herdsman-converters/converters/toZigbee.js:1326:20)
    at EntityPublish.onMQTTMessage (/opt/zigbee2mqtt/lib/extension/publish.js:197:52)
    at Controller.callExtensionMethod (/opt/zigbee2mqtt/lib/controller.js:352:44)
    at Controller.onMQTTMessage (/opt/zigbee2mqtt/lib/controller.js:246:14)
    at MQTT.emit (events.js:198:13)
    at MQTT.onMessage (/opt/zigbee2mqtt/lib/mqtt.js:95:14)
    at MqttClient.emit (events.js:198:13)
    at MqttClient._handlePublish (/opt/zigbee2mqtt/node_modules/mqtt/lib/client.js:1269:12)
    at MqttClient._handlePacket (/opt/zigbee2mqtt/node_modules/mqtt/lib/client.js:409:12)
    at work (/opt/zigbee2mqtt/node_modules/mqtt/lib/client.js:320:12)
```

Reply:
I made a small typo `commandReponse` -> `commandResponse`

Reply:
Oh god, i didn't saw that. Now it throws no error.
The problem is, i think the device isn't correctly paired to zigbee2mqtt because the 3 buttons light up alternately. When i pair the device to the original base, only the home button lights up. When i switch the mode to away, the away button lights up. But that doesn't work when its paired to zigbee2mqtt.
I captured the pairing process with the original base. @Koenkk, could you look into this?

Reply:
Can you share the network key of the sniff with the original base?

Reply:
Sure, Trust Center link key is the same as in your how to and network key is `64521b26239adcc71c3aac2579dba11d`

Reply:
I don't think this is the whole pairing process, can you start sniffing before you pair it?

Reply:
Thank you for your time. I paired again, here is the log.

Reply:
Thanks, I've investigated but cannot spot the issue, could you make the same sniff when pairing with zigbee2mqtt?

Reply:
Here is the sniff when pairing with zigbee2mqtt:


and here is the zigbee2mqtt log:
[https://pastebin.com/g6UiSmRb](https://pastebin.com/g6UiSmRb)

Reply:
Can you try to add to `/app/zigbee2mqtt/node_modules/zigbee-herdsman/dist/controller/event.js` `CommandsLookup`: `'getPanelStatus': 'commandGetPanelStatus',`.

Now change the definition to:

```js
{
    zigbeeModel: ['3400-D'],
    model: '3400-D',
    vendor: 'Centralite',
    description: '3400-D keypad',
    supports: 'action, arm',
    meta: {configureKey: 1, disableDefaultResponse: true},
    fromZigbee: [ fz.3400_command_arm, fz.temperature ],
    toZigbee: [ tz.arm_mode ],
    configure: async (device, coordinatorEndpoint) => {
        const endpoint = device.getEndpoint(1);
        await bind(endpoint, coordinatorEndpoint, ['msTemperatureMeasurement', 'genPowerCfg', 'ssIasZone', 'ssIasAce']);
        await configureReporting.temperature(endpoint);
        await configureReporting.batteryVoltage(endpoint);
    },
    onEvent: async (type, data, device) => {
        console.log('\nGOT MESSAGE: ' + data + ' \n')
        if (type === 'message' && data.type === 'getPanelStatus' && data.cluster === 'ssIasAce') {
            const payload = {panelstatus: 0x80, secondsremain: 0x00, audiblenotif: 0x00, alarmstatus: 0x00};
            await device.getEndpoint(1).commandResponse('ssIasAce', 'getPanelStatusRsp', payload, {}, data.meta.zclTransactionSequenceNumber);
        }
    }
}
```

Now repair the device and provide both the Zigbee2mqtt logging and sniff here.

Reply:
I've edited `/app/zigbee2mqtt/node_modules/zigbee-herdsman/dist/controller/event.js` and the device defintion.
I've repaired the device. Here is the sniff:


and here the log:
https://pastebin.com/szACU0pP

Reply:
Can you try with:

```js
{
    zigbeeModel: ['3400-D'],
    model: '3400-D',
    vendor: 'Centralite',
    description: '3400-D keypad',
    supports: 'action, arm',
    meta: {configureKey: 1, disableDefaultResponse: true},
    fromZigbee: [ fz.3400_command_arm, fz.temperature ],
    toZigbee: [ tz.arm_mode ],
    configure: async (device, coordinatorEndpoint) => {
        const endpoint = device.getEndpoint(1);
        await bind(endpoint, coordinatorEndpoint, ['msTemperatureMeasurement', 'genPowerCfg', 'ssIasZone', 'ssIasAce']);
        await configureReporting.temperature(endpoint);
        await configureReporting.batteryVoltage(endpoint);
    },
    onEvent: async (type, data, device) => {
        console.log('\nGOT MESSAGE: ' + JSON.stringify(data) + ' \n')
        if (type === 'message' && data.type === 'commandGetPanelStatus' && data.cluster === 'ssIasAce') {
            const payload = {panelstatus: 0x80, secondsremain: 0x00, audiblenotif: 0x00, alarmstatus: 0x00};
            await device.getEndpoint(1).commandResponse('ssIasAce', 'getPanelStatusRsp', payload, {}, data.meta.zclTransactionSequenceNumber);
        }
    }
}
```

Reply:
Here is the sniff:


and here the log:
https://pastebin.com/5af099Kj

Reply:
Looks good now, does it pair correclty now?

Reply:
Looks better now.
When i want to disarm by keypad and then send manually the payload `{"disarm":""}` to  the keypad `zigbee2mqtt/0x000d6f000d8a5f63/set` the keypad doesnt set the mode to disarmed.

Can you have a look at the log please?

[zigbee2mqtt log](https://pastebin.com/ujShqDmY)

and the zigbee2mqtt sniff:

Reply:
Looking at https://github.com/Koenkk/zigbee-herdsman-converters/blob/0f8d0cc9406c938902d1f851c1f759014e6cb35d/converters/toZigbee.js#L2130, I think you should send `{"arm_mode": "disarm"}`

Reply:
> Change the converter to 
> 
> ```js
> 3400_arm_mode: {
>     key: ['disarm],
>     convertSet: async (entity, key, value, meta) => {
>         entity.commandReponse('ssIasAce', 'armRsp', {armnotification: 0})
>     },
> },    
> ```
> 
> Now you can send the command by sending to topic `zigbee2mqtt/[DEVICE_FRIENDLY_NAME]/set` with payload `{"disarm":""}`

You posted this a few days ago and my tozigbee looks exactly like this.
Can you please tell me how tozigbee should look like?

Reply:
Looking at https://github.com/Koenkk/zigbee-herdsman-converters/pull/1301#discussion_r441142739, you cannot send a disarm yourself, the device first send a disarm request and then you have to respond to it.

This can be done with the following definition, once a disarm request is received, zigbee2mqtt automatically sends the response.

```js
{
    zigbeeModel: ['3400-D'],
    model: '3400-D',
    vendor: 'Centralite',
    description: '3400-D keypad',
    supports: 'action, arm',
    meta: {configureKey: 1, disableDefaultResponse: true},
    fromZigbee: [ fz.3400_command_arm, fz.temperature ],
    toZigbee: [ tz.arm_mode ],
    configure: async (device, coordinatorEndpoint) => {
        const endpoint = device.getEndpoint(1);
        await bind(endpoint, coordinatorEndpoint, ['msTemperatureMeasurement', 'genPowerCfg', 'ssIasZone', 'ssIasAce']);
        await configureReporting.temperature(endpoint);
        await configureReporting.batteryVoltage(endpoint);
    },
    onEvent: async (type, data, device) => {
        console.log('\nGOT MESSAGE: ' + JSON.stringify(data) + ' \n')
        if (type === 'message' && data.type === 'commandGetPanelStatus' && data.cluster === 'ssIasAce') {
            const payload = {panelstatus: 0x80, secondsremain: 0x00, audiblenotif: 0x00, alarmstatus: 0x00};
            await device.getEndpoint(1).commandResponse('ssIasAce', 'getPanelStatusRsp', payload, {}, data.meta.zclTransactionSequenceNumber);
        }

        if (type === 'message' && data.type === 'commandArm' && data.cluster === 'ssIasAce' && data.data.armmode === 0) {
            const payload = {armnotification: 0};
            await device.getEndpoint(1).commandResponse('ssIasAce', 'armRsp', payload, {}, data.meta.zclTransactionSequenceNumber);
        }
    }
}

```

Reply:
Thank you for this.
I think zigbee2mqtt cant respond directly at receiving the command from the keypad because the keycode has to be checked if its correct first.
But for debugging it is ok that zigbee2mqtt responds directly.
I've editted my device.js file accordingly to your last post. The keypad reacts differently as before. I think it accepts the command from z2m but doesnt change the mode to disarm (it is in away mode and stays there)
I've uploaded the log from z2m here:
[https://pastebin.com/X2FK4d8K](https://pastebin.com/X2FK4d8K)

Reply:
Can you capture the disarm process with both the original gateway and zigbee2mqtt and share those?

Reply:
Disarm process with original gateway without using keypad (disarmed with app from armed away to disarm):


Disarm process with original gateway using keypad (from armed away to disarm):


Disarm process with zigbee2mqtt using keypad (keypad is in state armed away. from armed away to disarm):

Reply:
Seems we need to send an additional command (according to groovy script and sniff), please try with below and provide sniff again when unlocking from keypad with z2m (and let me know if works ofcourse :))

```js
{
    zigbeeModel: ['3400-D'],
    model: '3400-D',
    vendor: 'CentraLite',
    description: '3400-D Keypad',
    supports: 'action, arm',
    meta: {configureKey: 1, disableDefaultResponse: true},
    fromZigbee: [ fz.command_arm, fz.temperature ],
    toZigbee: [ tz.centralite_arm_mode],
    configure: async (device, coordinatorEndpoint) => {
        const endpoint = device.getEndpoint(1);
        await bind(endpoint, coordinatorEndpoint, ['msTemperatureMeasurement', 'genPowerCfg', 'ssIasZone', 'ssIasAce']);
        await configureReporting.temperature(endpoint);
        await configureReporting.batteryVoltage(endpoint);
    },
    onEvent: async (type, data, device) => {
        console.log('\nGOT MESSAGE: ' + JSON.stringify(data) + ' \n')
        if (type === 'message' && data.type === 'commandGetPanelStatus' && data.cluster === 'ssIasAce') {
            const payload = {panelstatus: 0x80, secondsremain: 0x00, audiblenotif: 0x00, alarmstatus: 0x00};
            await device.getEndpoint(1).commandResponse('ssIasAce', 'getPanelStatusRsp', payload, {}, data.meta.zclTransactionSequenceNumber);
        }
        if (type === 'message' && data.type === 'commandArm' && data.cluster === 'ssIasAce' && data.data.armmode === 0) {
            await device.getEndpoint(1).commandResponse('ssIasAce', 'armRsp', {armnotification: 0}, {}, data.meta.zclTransactionSequenceNumber);
            await device.getEndpoint(1).commandResponse('ssIasAce', 'panelStatusChanged', {panelstatus: 0, secondsremain: 0, audiblenotif: 0, alarmstatus: 0}, {disableDefaultResponse: true}, data.meta.zclTransactionSequenceNumber);
        }
    }
}
```

Reply:
Hi, i can ckeck this in one week. I'm on vacation right now.

Reply:
Enjoy your holiday!

Reply:
I've captured the unlocking process with z2m again. LED on keypad moves now from away to home but after 5 seconds back to away.

Reply:
Did a potential fix in the latest dev, can you check?

Changes will be available in the latest dev branch in a few hours (https://www.zigbee2mqtt.io/how_tos/how-to-switch-to-dev-branch.html)

Reply:
The keypad stays now in mode 'unarmed' when i unarm it from the keypad.
Can you please implement mode 'arm_all_zones'? We can switch then between 'unarmed' and 'all_zones_armed'. When i switch from 'unarmed' to 'arm_all_zones' now, all three lights on the keypad lights up successively. When i switch back to 'unarmed', only 'unarmed' lights up.

I've captured disarm process:

and i've captured the arm_all_zones process:

Reply:
Try with:

```js
{
    zigbeeModel: ['3400-D'],
    model: '3400-D',
    vendor: 'CentraLite',
    description: '3400-D Keypad',
    supports: 'action, arm',
    meta: {configureKey: 1, disableDefaultResponse: true},
    fromZigbee: [ fz.command_arm, fz.temperature ],
    toZigbee: [ tz.centralite_arm_mode],
    configure: async (device, coordinatorEndpoint) => {
        const endpoint = device.getEndpoint(1);
        await bind(endpoint, coordinatorEndpoint, ['msTemperatureMeasurement', 'genPowerCfg', 'ssIasZone', 'ssIasAce']);
        await configureReporting.temperature(endpoint);
        await configureReporting.batteryVoltage(endpoint);
    },
    onEvent: async (type, data, device) => {
        console.log('\nGOT MESSAGE: ' + JSON.stringify(data) + ' \n')
        if (type === 'message' && data.type === 'commandGetPanelStatus' && data.cluster === 'ssIasAce') {
            const payload = {panelstatus: 0x80, secondsremain: 0x00, audiblenotif: 0x00, alarmstatus: 0x00};
            await device.getEndpoint(1).commandResponse('ssIasAce', 'getPanelStatusRsp', payload, {}, data.meta.zclTransactionSequenceNumber);
        }
        if (type === 'message' && data.type === 'commandArm' && data.cluster === 'ssIasAce') {
            await device.getEndpoint(1).commandResponse('ssIasAce', 'armRsp', {armnotification: data.data.armmode}, {}, data.meta.zclTransactionSequenceNumber);
            await device.getEndpoint(1).commandResponse('ssIasAce', 'panelStatusChanged', {panelstatus: 0, secondsremain: 0, audiblenotif: 0, alarmstatus: 0}, {disableDefaultResponse: true}, data.meta.zclTransactionSequenceNumber);
        }
    }
}
```

Reply:
With this code i cant pair the device to z2m. I get:

````
Zigbee2MQTT:error 2020-08-03 21:22:31: Failed to interview '0x000d6f000d8a5f63', device has not successfully been paired
````

I've captured the pairing process (tried to pair with z2m)

Reply:
Made some more improvements in the dev branch, can you check again with that?

Reply:
Thank you. I could pair the device again now.
I couldn't arm by keypaid. Here is the z2m sniff.

Reply:
Can you provide me the arm all zones with the original gateway?

Reply:
I've captured the arm process with the original gateway

Reply:
Try with:

```js
{
    zigbeeModel: ['3400-D'],
    model: '3400-D',
    vendor: 'CentraLite',
    description: '3400-D Keypad',
    supports: 'action, arm',
    meta: {configureKey: 1, disableDefaultResponse: true},
    fromZigbee: [ fz.command_arm, fz.temperature ],
    toZigbee: [ tz.centralite_arm_mode],
    configure: async (device, coordinatorEndpoint) => {
        const endpoint = device.getEndpoint(1);
        await bind(endpoint, coordinatorEndpoint, ['msTemperatureMeasurement', 'genPowerCfg', 'ssIasZone', 'ssIasAce']);
        await configureReporting.temperature(endpoint);
        await configureReporting.batteryVoltage(endpoint);
    },
    onEvent: async (type, data, device) => {
        console.log('\nGOT MESSAGE: ' + JSON.stringify(data) + ' \n');
        if (type === 'message' && data.type === 'commandGetPanelStatus' && data.cluster === 'ssIasAce') {
            const payload = {panelstatus: store[device.ieeeAddr] ? 0x80 : 0x00, secondsremain: 0x00, audiblenotif: 0x00, alarmstatus: 0x00};
            await device.getEndpoint(1).commandResponse('ssIasAce', 'getPanelStatusRsp', payload, {}, data.meta.zclTransactionSequenceNumber);
        }
        if (type === 'message' && data.type === 'commandArm' && data.cluster === 'ssIasAce') {
            await device.getEndpoint(1).commandResponse('ssIasAce', 'armRsp', {armnotification: data.data.armmode}, {}, data.meta.zclTransactionSequenceNumber);
            store[device.ieeeAddr] = data.data.armmode === 3; // 3 = arm, 0 = disarm
            await device.getEndpoint(1).commandResponse('ssIasAce', 'panelStatusChanged', {panelstatus: store[device.ieeeAddr] ? 0x80 : 0x00, secondsremain: 0, audiblenotif: 0, alarmstatus: 0}, {disableDefaultResponse: true}, data.meta.zclTransactionSequenceNumber);
        }
    }
}
```

Reply:
Perfect, you've got it. We can now switch from disarmed to arm_all_zones and back.
The keypad has a third function button. In my version of the device it is a bell. I've captured the process (pressed the bell and entered the 6 digit code when device is in unarmed mode) I think we can use this mode when it is night, we stay at home and want to arm the system. Can you please implement this funtion?

pressed bell button when device is in unarmed mode:


![Centralite 3400-D](https://cdn.smarthome.de/product-images/358b7b65-557e-4aed-9cc2-94764e16162b/default/centralite-security-tastenfeld-weiss.png)

Reply:
I've finalised the code & docs. First make sure you are on a clean latest dev branch and apply the changes from: https://github.com/Koenkk/zigbee-herdsman-converters/pull/1301/files

Documentation can be found here: https://github.com/Koenkk/zigbee2mqtt.io/blob/centralite_3400d/docs/devices/3400_d.md

Reply:
> I've finalised the code & docs. First make sure you are on a clean latest dev branch and apply the changes from: https://github.com/Koenkk/zigbee-herdsman-converters/pull/1301/files

Can you please tell me how to appy the changes from the files?

Reply:
How are you  running z2m? (hassio/docker/bare metal?)

Reply:
I'm running it on bare metal.

Reply:
I'm running it on bare metal.

Reply:
In that case execute the following commands:

```
cd /app/zigbee2mqtt/node_modules
rm -rf zigbee-herdsman-converters
git clone https://github.com/sparky3387/zigbee-herdsman-converters.git
cd zigbee-herdsman-converters
npm ci
```

Now changes are applied and you can start zigbee2mqtt

Reply:
Thank you for your help. I could pull the latest commits for the dev branch and applied the changes.
I could pair the device and can manually switch the modes by sending mqtt messages to the channel.
Does it make sense to publish the value of ``action_transaction`` to home assistant via the ``mappings`` in ``lib/extension/homeassistant.js``? We can then use the value in an home assistant automation to answer accordingly when we use the keypad to (dis)arm the system. Can we publish the curent arm state, too?
What should we else publish from the device to home assistant?

Reply:
I plan on adding an action sensor for this, via the attributes of that it is possible to access the action_transaction. 

The mapping will be this:

```js
'3400-D': [cfg.sensor_click, cfg.sensor_battery, cfg.sensor_action],
```

Note that this is kinda a strange the device, the state should be kept by Home Assistant as the device has no state! (it can only request for state changes, wether to apply them is up the Home Assistant)

Reply:
Right now i'am using home assistant to collect the state change via mqtt sensor. 
z2m publishes to zigbee2mqtt/0x000d6f000d8a5f63 the state change but it does this two times:

````
Zigbee2MQTT:info  2020-08-10 20:02:56: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63', payload '{"linkquality":2,"temperature":32.28,"keycode":"\u0000\u0000\u0000\u0000\u0000\u0000","voltage":3000,"action":"disarm","action_code":"112233","action_zone":0,"action_transaction":61}'
Zigbee2MQTT:info  2020-08-10 20:02:56: MQTT publish: topic 'zigbee2mqtt/0x000d6f000d8a5f63', payload '{"linkquality":2,"temperature":32.28,"keycode":"\u0000\u0000\u0000\u0000\u0000\u0000","voltage":3000,"action":""}'
````

Here z2m overwrites the first "action":"disarm" with an empty "action":"". Often home assistant doens't catch the first message. Can we get arround this?

Reply:
This is expected as it will trigger a state change in the action which can be used in an automation (https://www.zigbee2mqtt.io/integration/home_assistant.html#via-home-assistant-entity). If that doesn't work try via MQTT: https://www.zigbee2mqtt.io/integration/home_assistant.html#via-mqtt

Reply:
Sorry for the delay. Thank you for the links, i've got it working with the mqtt method.
Can you please implement one more thing? We have to manage wrong keypad entry codes. I've typed in a wrong entry code and attached the sniff with the original gateway:

Reply:
With the latest code you can now send `invalid_code` as `mode`, updated https://github.com/Koenkk/zigbee2mqtt.io/blob/centralite_3400d/docs/devices/3400_d.md

Reply:
Thank you, it works!
When i pair the keypad with z2m i get many 'no converter available' messages:

````
Zigbee2MQTT:debug 2020-08-24 20:30:04: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'genBasic', data '{"modelId":"3400-D"}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:04: No converter available for '3400-D' with cluster 'genBasic' and type 'readResponse' and data '{"modelId":"3400-D"}'
Zigbee2MQTT:debug 2020-08-24 20:30:05: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'genBasic', data '{"manufacturerName":"CentraLite"}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:05: No converter available for '3400-D' with cluster 'genBasic' and type 'readResponse' and data '{"manufacturerName":"CentraLite"}'
Zigbee2MQTT:debug 2020-08-24 20:30:06: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'genBasic', data '{"powerSource":3}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:06: No converter available for '3400-D' with cluster 'genBasic' and type 'readResponse' and data '{"powerSource":3}'
Zigbee2MQTT:debug 2020-08-24 20:30:06: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'genBasic', data '{"powerSource":3}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:06: No converter available for '3400-D' with cluster 'genBasic' and type 'readResponse' and data '{"powerSource":3}'
Zigbee2MQTT:debug 2020-08-24 20:30:08: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'genBasic', data '{"zclVersion":1}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:08: No converter available for '3400-D' with cluster 'genBasic' and type 'readResponse' and data '{"zclVersion":1}'
Zigbee2MQTT:debug 2020-08-24 20:30:09: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:09: No converter available for '3400-D' with cluster 'genBasic' and type 'readResponse' and data '{}'
Zigbee2MQTT:debug 2020-08-24 20:30:10: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'genBasic', data '{"stackVersion":2}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:10: No converter available for '3400-D' with cluster 'genBasic' and type 'readResponse' and data '{"stackVersion":2}'
Zigbee2MQTT:debug 2020-08-24 20:30:11: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'genBasic', data '{"hwVersion":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:11: No converter available for '3400-D' with cluster 'genBasic' and type 'readResponse' and data '{"hwVersion":0}'
Zigbee2MQTT:debug 2020-08-24 20:30:12: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:12: No converter available for '3400-D' with cluster 'genBasic' and type 'readResponse' and data '{}'
Zigbee2MQTT:debug 2020-08-24 20:30:13: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'genBasic', data '{}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:13: No converter available for '3400-D' with cluster 'genBasic' and type 'readResponse' and data '{}'
Zigbee2MQTT:debug 2020-08-24 20:30:14: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0xffffffffffffffff","zoneState":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:14: No converter available for '3400-D' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0xffffffffffffffff","zoneState":0}'
Zigbee2MQTT:debug 2020-08-24 20:30:16: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:16: No converter available for '3400-D' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}'
Zigbee2MQTT:debug 2020-08-24 20:30:17: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:17: No converter available for '3400-D' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}'
Zigbee2MQTT:debug 2020-08-24 20:30:18: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:18: No converter available for '3400-D' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}'
Zigbee2MQTT:debug 2020-08-24 20:30:19: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:19: No converter available for '3400-D' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}'
Zigbee2MQTT:debug 2020-08-24 20:30:20: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:20: No converter available for '3400-D' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}'
Zigbee2MQTT:debug 2020-08-24 20:30:21: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:21: No converter available for '3400-D' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}'
Zigbee2MQTT:debug 2020-08-24 20:30:22: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:22: No converter available for '3400-D' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}'
Zigbee2MQTT:debug 2020-08-24 20:30:24: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:24: No converter available for '3400-D' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}'
Zigbee2MQTT:debug 2020-08-24 20:30:25: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:25: No converter available for '3400-D' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}'
Zigbee2MQTT:debug 2020-08-24 20:30:26: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:26: No converter available for '3400-D' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}'
Zigbee2MQTT:debug 2020-08-24 20:30:27: Received Zigbee message from '0x000d6f000d8a5f63', type 'readResponse', cluster 'ssIasZone', data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}' from endpoint 1 with groupID 0
Zigbee2MQTT:debug 2020-08-24 20:30:27: No converter available for '3400-D' with cluster 'ssIasZone' and type 'readResponse' and data '{"iasCieAddr":"0x00124b00194a208e","zoneState":0}'
````

Can we fix this?

Reply:
This is expected, it's just a notice that the message will not be converted to MQTT (it's fine as it's not intereseting). Therefore it's logged as `debug`.